### PR TITLE
added comments to contact.js

### LINKED ISSUE #48: Fix : Added Comments for contact.js
adding comments to contact.js file

### PR DESCRIPTION
**Pull request by OmarBassatni97**

### CODE REVIEW

File: src/services/contact.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+    e.preventDefault(); //to stop the form from reloading the page onSubmit
 
-    emailjs.sendForm('service_iwqsu3f', 'template_d0s4lc6', templateParams, 'SMrzCAgEF5PcNX7Jo')
+    emailjs.sendForm('service_iwqsu3f', 'template_d0s4lc6', templateParams, 'SMrzCAgEF5PcNX7Jo') //emailjs method to send the form that takes the form inputs as a third argument
 
-    e.target.reset()
+    e.target.reset() //reset the form after the submit
 };
 
 export { mailSender }
```
Comment (Iseluin) [inline src/services/contact.js]: Please structure your comments like this:

`//comment`
`...actual code`

**Review by Iseluin — CHANGES_REQUESTED** [review]
Not many comments and they are on point, good job.
Please structure your issues like the existing ones next time you open issues.
There is a little review I left, please change that and we are good to go!

File: src/services/contact.js (hunk 1 above)
Comment (OmarBassatni97) [reply src/services/contact.js]: you want me to write the word comment on a separate line before each comment?

Comment (Iseluin) [reply src/services/contact.js]: nope, move your comments one line up from the line of code that it is relevant of. They should not be in the same line.

**Review by Iseluin — APPROVED** [review]
👍🏻

**Pull request merged**